modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm

```diff
@@ -550,6 +550,198 @@ sub sift_score {
     return $self->{sift_score};
 }
 
+=head2 dbnsfp_cadd_prediction
+
+  Description: Return the qualitative CADD prediction for the effect of this allele.
+               (Note that we currently only have predictions for variants that 
+               result in single amino acid substitutions in human)
+  Returntype : string (one of 'observed', 'simulated') if this is a non-synonymous 
+               mutation and a prediction is available, undef otherwise. Predictions
```

Comment:
Can we use 'missense change' rather than non-synonymous mutation' here as we normally do.